### PR TITLE
Add playlists

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request I want to :
1.Add playlist's components.

### CODE REVIEW

File: src/components/PlayLists/index.jsx [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 import "./styles.css";
 
 const PlayLists = () => {
-  return <p>Playlists</p>;
+  const playlistsItems = playlistsData.playlists.items;
+  const playlists = playlistsItems.map((playlistsItem) => (
+    <PlayListsItem details={playlistsItem} />
+  ));
+
+  return <ul class="playlists">{playlists}</ul>;
```
Comment (blacksrc) [inline src/components/PlayLists/index.jsx]: No need to define a variable. Let the ui part remain in the return unless it's necessary.
```suggestion
  return <ul class="playlists">{playlistsItems.map((playlistsItem) => (
    <PlayListsItem details={playlistsItem} />
  ))}</ul>;
```

File: src/components/PlayLists/index.jsx [hunk 2]
```diff
@@ -1,7 +1,14 @@
+import PlayListsItem from "../PlayListItem/index.jsx";
+import playlistsData from "../../../assests/json/playlists.json";
```
Comment (blacksrc) [inline src/components/PlayLists/index.jsx]: Please, move the `assets` directory into the `src`.

Comment (mim24sd) [reply src/components/PlayLists/index.jsx]: doneeee

File: src/components/PlayLists/index.jsx (hunk 1 above)
Comment (mim24sd) [reply src/components/PlayLists/index.jsx]: doneeeeeee

**Pull request merged**